devtools/datasette/publish.py

```diff
@@ -82,9 +83,78 @@ def inspect_data(datasets: list[str], pudl_out: Path) -> str:
     return inspect_output
 
 
-def metadata(pudl_out: Path) -> str:
+def check_tables_have_metadata(
```

Comment:
This checks all tables in the databases have metadata. This feels like a check that should be available in datasette. I left a [question on the datasette github](https://github.com/simonw/datasette/discussions/2233).

Reply:
I think we should add this test to `test/integration/datasette_metadata_test.py` so that we catch new errors in CI rather than only in the nightly builds. Is there some reason that won't work? Maybe create a helper function that can be used both in the tests and here.

Reply:
That makes sense. I'll make the change.